Dockerfile

```diff
@@ -92,12 +92,12 @@ RUN chgrp -R 0 ${AMBASSADOR_ROOT} && \
     chmod -R g=u ${AMBASSADOR_ROOT} /etc/passwd
 
 # COPY the entrypoint and Python-kubewatch and make them runnable.
-COPY ambassador/entrypoint.sh .
-COPY ambassador/grab-snapshots.py .
-COPY ambassador/kick_ads.sh .
-COPY ambassador/kubewatch.py .
-COPY ambassador/post_update.py .
-COPY ambassador/watch_hook.py .
+COPY python/entrypoint.sh .
```

Comment:
The wholesale rename of `ambassador` to `python` produces a couple of really odd effects. This is one of thing: `entrypoint.sh` isn't Python code, so it seems strange that it would be in a directory named `python`. 

Another effect - and I think this is more of a real issue - is that we're almost certainly shipping more than we should be now. `kat` actually is not code that we need to ship to end users -- developers need it, but it doesn't need to be in the Docker images we ship. (Though having `mypy` type-check `kat` is a huge win...)

Reply:
I agree that `entrypoint.sh` should be in a difference place. We are following the [Go Standard Project](https://github.com/golang-standards/project-layout) layout. Maybe we should move it to `/build` or `/deployments` or /configs`...

Reply:
Regarding `kat`, we decided that, even when this package is used only by developers, it didn' hurt to include it in the final image...